RecoTracker/MkFitCore/src/MkFinder.cc

```diff
@@ -996,6 +998,25 @@ namespace mkfit {
               isCompatible &= passStripChargePCMfromTrack(
                   itrack, layer_of_hits.is_barrel(), charge_pcm[itrack], Hit::minChargePerCM(), propPar, m_msErr);
             }
+            // Select only SiStrip hits with cluster size < maxClusterSize
+            if (!layer_of_hits.is_pixel()) {
+              if (layer_of_hits.refHit(m_XHitArr.At(itrack, hit_cnt, 0)).spanRows() >=
+                  m_iteration_params->maxClusterSize) {
+                isTooLargeCluster[itrack] = true;
+                isCompatible = false;
+              }
+            }
+            // Uncomment to apply analogous cut on cluster size of SiPixel hits
+            //else {
```

Comment:
the usual comment applies about turning this into an issue (if it's a future item), or removing the commented out code.

Reply:
Removed.